scripts/get-marker-address.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# --- Common Binary and Home Directory ---
+SIMD_BIN="./simapp/build/simd"
+HOME_DIR="--home ./.vaulty" # Ensure this is consistent with your setup
+
+# --- Script Logic ---
+
+# Check if the denom argument was provided
+if [ -z "$1" ]; then
+  echo "Usage: $0 <denom>"
+  echo "Example: $0 hotdogcoin"
+  exit 1
+fi
+
+# Assign the first argument to the DENOM variable
+DENOM="$1"
+
+MARKER_ADDRESS=$($SIMD_BIN q marker get "$DENOM" $HOME_DIR 2>&1 | grep 'base_account' -A 2 | grep 'address' | awk '{ print $2 }' | tr -d '\r')
```

Comment:
If you are ok with a jq dependency you could directly address this value from the response.

Reply:
@Taztingo Let's get this in the next PR.  We can update our scripts to a `-o json` too.

Reply:
Not sure it’s worth the effort. Just pointing it out if we have a more complex one in the future.